src/Agent.Worker/TaskRunner.cs

```diff
@@ -564,5 +568,31 @@ private void PrintTaskMetaData(Definition taskDefinition)
             ExecutionContext.Output($"Help         : {taskDefinition.Data.HelpUrl ?? taskDefinition.Data.HelpMarkDown}");
             ExecutionContext.Output("==============================================================================");
         }
+        private void PublishTelemetry(Definition taskDefinition, HandlerData handlerData)
+        {
+            ArgUtil.NotNull(Task, nameof(Task));
+            ArgUtil.NotNull(Task.Reference, nameof(Task.Reference));
+            ArgUtil.NotNull(taskDefinition.Data, nameof(taskDefinition.Data));
+
+            var useNode10 = AgentKnobs.UseNode10.GetValue(ExecutionContext).AsString();
+            Dictionary<string, string> telemetryData = new Dictionary<string, string>();
```

Comment:
Maybe it's need extra blank line before this line

Reply:
Done